packages/lesswrong/components/posts/RecombeePostsList.tsx

```diff
@@ -96,10 +99,11 @@ export const RecombeePostsList = ({ algorithm, settings, limit = 15, classes }:
 
   const results: RecommendedPost[] | undefined = data?.[resolverName]?.results;
   const postIds = results?.map(({post}) => post._id) ?? [];
+  const postIdsWithScenario = results?.map(({post, scenario}) => ({postId: post._id, scenario: scenario ?? 'none'})) ?? [];
 
   useOnMountTracking({
     eventType: "postList",
-    eventProps: { postIds },
+    eventProps: { postIds, postIdsWithScenario, algorithm },
```

Comment:
Do we need to keep `postIds` around?  Backward compatibility issues to drop it if the queries we have relying on it are annoying to rewrite, but the additional payload size is non-trivial...